cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -169,6 +182,93 @@ protected void setMetricsEnabled(boolean isMetricsEnabled) {
         this.isMetricsEnabled = isMetricsEnabled;
     }
 
+    protected String getLogConfig() {
+        return logConfig;
+    }
+
+    private Map getDefaultLogConfig() {
+        Map<String, String> defaultSettings = new LinkedHashMap<>();
```

Comment:
I'm quite sure that @tombentley will ask why are you using `LinkedHashMap` and not properties :-)

Reply:
I was not sure if Properies keeps order of data.

Reply:
That is interesting question ... I'm not sure they do. But is it important for the log4j stuff? The file is called `.properties` so I thought it will be compatible. Anyway ... if that's the case than feel free to ignore this comment.

Reply:
Well, in origin .properties file is this substitution:
```
kafka.root.logger.level=INFO
log4j.rootLogger=${kafka.root.logger.level}, CONSOLE
```
I am not sure, what happens in log4j, when the value is first used an then defined. But we can get rid of this issue by merging these two lines.
My opinion is properties does not preserve the order, because `Properties extends Hashtable`.

Reply:
Yeah, my memory from writing the tests for the configuration field also tells me that it doesn't preserve order. Because AFAIK it always reordered the options.